IBX-181: Rewrite query and controller documentation (#1370)
ezsystems/developer-documentation: docs/guide/content_rendering/queries_and_controllers/controllers.md

# Controllers

> Configuring a controller enables you to modify and enhance the way in which the built-in content view controller renders content.

Reviewer: ```suggestion
By configuring a controller you can modify and enhance the way in which the built-in content view controller renders content.
```

You indicate which controller to use in the [content view configuration](../templates/template_configuration.md), under the `controller` key:

``` yaml
[[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 23, 26) =]][[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 28, 30) =]]
```

``` php

> [[= include_file('code_samples/front/embed_content/src/Controller/RelationController.php', 2, 10) =]]

Author: ```suggestion
[[= include_file('code_samples/front/embed_content/src/Controller/RelationController.php', 2, 9) =]]
```

# Controllers

Configuring a controller enables you to modify and enhance the way in which the built-in content view controller renders content.

You indicate which controller to use in the [content view configuration](../templates/template_configuration.md), under the `controller` key:

``` yaml
[[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 23, 26) =]][[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 28, 30) =]]
```

> A custom controller must extend `eZ\Bundle\EzPublishCoreBundle\Controller`.

Reviewer 2: Is this really true?

Author: Ah, I was hoping you'd know ;)

Reviewer 2: I think it is not needed but let me check then :D

Reviewer 2: @DominikaK 
I now can confirm, that this controller does not have to extend anything in order to this to work.
